renderers/games/game.py

```diff
@@ -46,27 +47,26 @@ def render_live_game(canvas, layout: Layout, colors: Color, scoreboard: Scoreboa
 
 
 # --------------- at-bat ---------------
-def _render_at_bat(canvas, layout, colors, atbat: AtBat, text_pos, strikeout, looking, animation, pitches: Pitches):
+def _render_at_bat(canvas, layout, colors, atbat: AtBat, text_pos, play_result, animation, pitches: Pitches):  
     plength = __render_pitcher_text(canvas, layout, colors, atbat.pitcher, pitches, text_pos)
     __render_pitch_text(canvas, layout, colors, pitches)
     __render_pitch_count(canvas, layout, colors, pitches)
-    if strikeout:
+    if play_result in PLAY_RESULT_UPDATES:
         if animation:
-            __render_strikeout(canvas, layout, colors, looking)
+            __render_play_result(canvas, layout, colors, play_result)
         return plength
     else:
         blength = __render_batter_text(canvas, layout, colors, atbat.batter, text_pos)
         return max(plength, blength)
 
-
-def __render_strikeout(canvas, layout, colors, looking):
-    coords = layout.coords("atbat.strikeout")
-    color = colors.graphics_color("atbat.strikeout")
-    font = layout.font("atbat.strikeout")
-    text = "ꓘ" if looking else "K"
+def __render_play_result(canvas, layout, colors, play_result):
+    
+    coords = layout.coords("atbat.play_result")
+    color = colors.graphics_color("atbat.play_result")
+    font = layout.font("atbat.play_result")
+    text = PLAY_RESULTS[play_result][coords["desc_length"]]
```

Comment:
This is a brittle lookup, it will fail if `play_result` or the result of `coords["desc_length"]` can't be found

Lots of ways to handle it, but easiest is `try`/`catch` on `KeyError`

```suggestion
    try:
      text = PLAY_RESULTS[play_result][coords["desc_length"]]
    except KeyError:
      return # There's no text or coordinates to render
```